{"name": "Ошибка при создании без логина", "status": "passed", "description": "Проверка, что при отсутствии логина возвращается ошибка 400", "start": 1752323985539, "stop": 1752323985799, "uuid": "6104e466-e01e-4e5c-8a6b-c1961782f0f7", "historyId": "0ef99d4f0173f8c9a605f86ee8a0411c", "testCaseId": "0ef99d4f0173f8c9a605f86ee8a0411c", "fullName": "Tests.test_create_courier.TestCreateCourier#test_error_without_login", "labels": [{"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "DESKTOP-T2S04D8"}, {"name": "thread", "value": "80-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_create_courier"}]}